src/bpm/runc/adapter/adapter_test.go

```diff
@@ -202,16 +202,18 @@ var _ = Describe("RuncAdapter", func() {
 					"RAVE": "true",
 					"ONE":  "two",
 				},
+				EphemeralDisk:  true,
+				PersistentDisk: false,
 				AdditionalVolumes: []config.Volume{
 					{Path: "/path/to/volume/1", Writable: true},
 					{Path: "/path/to/volume/jna-tmp", Writable: true, AllowExecutions: true},
-					// Testing duplicate volumes
+					// Duplicate volumes
 					{Path: "/path/to/volume/2"},
 					{Path: "/path/to/volume/2"},
-					// Testing duplicate store mount and ignore writable flag
+					// Duplicate data mount
 					{Path: bpmCfg.DataDir()},
-					// Testing duplicate data mount
-					{Path: bpmCfg.StoreDir(), Writable: true},
+					// Testing store mount override
+					{Path: bpmCfg.StoreDir(), Writable: true, AllowExecutions: true},
```

Comment:
This was somewhat deliberate. Id' rather not have this package depend on a completely different package (`config`) to sanitize its inputs because the coupling isn't very obvious. At the same time I didn't want this to need to duplicate the configuration validation here. So I decided to see what it would be like to have overriding the mount be valid in the adapter and have it only complain on practically invalid configurations e.g. two volumes mounted in the same place.

Reply:
Didn't notice that we already had a test that checks `PersistentDisk: true`. Makes sense to me.